kuka_kr5_support/launch/test_kr5_arc.launch.py

```diff
@@ -0,0 +1,141 @@
+import os
+
+from ament_index_python.packages import get_package_share_directory
+
+from launch import LaunchDescription
+from launch.actions import DeclareLaunchArgument, RegisterEventHandler
+from launch.event_handlers import OnProcessExit
+from launch.substitutions import Command, FindExecutable, LaunchConfiguration, PathJoinSubstitution
+
+from launch_ros.actions import Node
+from launch_ros.substitutions import FindPackageShare
+
+import xacro
+
+
+def generate_launch_description():
```

Comment:
I see here now the difference between load and test. Before the packages were not starting ros_control at all.

So what we should do, is the following. Rename this file to `test_ros2_control_*`. Keep the above file and rename it to `test_*`, as described above.